debian/configure

```diff
@@ -120,7 +117,6 @@ KERNEL_DEPENDS_BUILD=libudev-dev
 KERNEL_HEADERS=
 KERNEL_VERSION=uspace
 MODUTILS_DEPENDS=
-EXTRA_FILES="usr/bin/rtapi_app"
 DISTRIB_NAME=uspace-$DISTRIB_NAME
 MAIN_PACKAGE_NAME=linuxcnc-uspace
 OTHER_MAIN_PACKAGE_NAME=linuxcnc
```

Comment:
If your already in and removing things, `MAIN_PACKAGE_NAME`, and `OTHER_MAIN_PACKAGE_NAME` is duplicated.